### PR TITLE
Day 2 steps 5-6 updates

### PR DESCRIPTION
**Pull request by ecraig12345**

### CODE REVIEW

File: step2-05/demo/README.md [hunk 1]
```diff
@@ -76,27 +76,25 @@ Given a state tree shaped like:
...
-We would organize our reducers matching the keys of the state tree:
+We would organize our reducers matching the keys of the state tree and combine them with [`combineReducers()`](https://redux.js.org/recipes/structuring-reducers/using-combinereducers):
 
 ```ts
 import { createReducer } from 'redux-starter-kit';
-import { combineReducer } from 'redux';
+import { combineReducers } from 'redux';
 
-const reducer = combineReducer({
-  // first argument: initial state
```
Comment (ecraig12345) [inline step2-05/demo/README.md]: Realized this comment distracts from the main point of this section, which is combineReducers

File: step2-06/demo/README.md [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
+# Step 2.6 - Redux: React Binding (Demo)
 
 [Lessons](../../) | [Exercise](../exercise/)
 
 Redux is currently the most popular Flux implementation, and the ecosystem of related libraries has grown as a result. This is one of the reasons why it is a very popular library within Microsoft products.
 
 Various GitHub users have collected "awesome lists" of tech and articles related to Redux. Here is [one such list](https://github.com/xgrommx/awesome-redux#react---a-javascript-library-for-building-user-interfaces), but it is literally impossible to list out all the related tech.
 
-In this step, we will continue with Redux. Step 2.5 code can be used with any other web frameworks. This step, we will hook up the Redux store with React components. The official way to do this is with the the [`react-redux`](https://react-redux.js.org/) library.
+In this step, we introduce one useful library that works with Redux: [`react-redux`](https://react-redux.js.org/). Whereas the Step 2.5 code could be used with any web UI framework, we'll now use `react-redux` to connect the store to our React app. There are two steps in this process:
```
Comment (ecraig12345) [inline step2-06/demo/README.md]: Reorganized to make this flow better from previous paragraph

File: step2-06/demo/README.md [hunk 3]
```diff
@@ -29,11 +27,11 @@ const App = () => {
 };
 ```
 
-## Mapping the Redux store to props
+## Mapping the store to props
```
Comment (ahtee) [inline step2-06/demo/README.md]: I dont mind keeping "Redux" here

File: step2-06/demo/README.md [hunk 4]
```diff
@@ -55,15 +53,15 @@ const ConnectedComponent = connect(
...
 
    - The first argument maps portions of the Redux _state tree_ into `<MyComponent>` _props_
-   - The second arguments maps dispatch functions into `<MyComponent>` _props_
+   - The second arguments maps dispatch functions into `<MyComponent>` _props_ (typically used as callbacks to respond to user interaction)
 
-3. Finally, `connect()` actually returns a function that **decorates** a `<MyComponent>` into `<ConnectedComponent>` - it is a strange syntax, so do study it more closely here.
+3. Finally, `connect()` actually returns a function, which we immediately call to **decorate** `<MyComponent>` into `<ConnectedComponent>` - it is a strange syntax, so do study it more closely here.
 
-> Yes, `connect()` is a function that takes in functions as arguments that returns a function that takes in a component that return a component. Try to say this fast 10 times :)
+> Yes, `connect()` is a function that takes in functions as arguments and returns a function that takes in a component and returns a component. Try to say this fast 10 times. :)
```
Comment (ahtee) [inline step2-06/demo/README.md]: `, and then returns` possibly?

**Pull request merged**